CNV/README.md

```diff
@@ -0,0 +1,10 @@
+# Container Native Virtualization Deployment
+
+## What does it do
+
+The script [deploy-cnv.sh](deploy-cnv.sh) deploys CNV through [OLM](https://github.com/operator-framework/operator-lifecycle-manager), meaning:
```

Comment:
Keep in mind this is CNV 2.0.   Might be worth making a note about it.

Reply:
Added. Thanks!